introduction.md

```diff
@@ -0,0 +1 @@
+Hi ! I'm COLUNDRUM and I do the Mastering GitHub of Code School for the fun ;)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.